ArduCopter/Log.cpp

```diff
@@ -324,6 +324,33 @@ void Copter::Log_Write_Heli()
     };
     logger.WriteBlock(&pkt_heli, sizeof(pkt_heli));
 }
+
+struct PACKED log_Heli_Swash {
+    LOG_PACKET_HEADER;
+    uint64_t time_us;
+    uint8_t  instance;
+    float    collective_angle;
+    float    cyclic_angle;
```

Comment:
```suggestion
    float    pitch_cyclic_angle;
    float    roll_cyclic_angle;
```
include pitch and roll cyclic values